scripts/dockerbot/check_prs.py

```diff
@@ -277,9 +313,19 @@ def check_status(pr):
                 logger.info('Setting task {} for pr {} to {}'
                             .format(task['stream'], pr['number'], states[task['result']]))
                 set_status(commit, states[task['result']], task['stream'])
+            task_states[task['stream']] = states[task['result']]
```

Comment:
>Even if the break is hit in 305, it's only going to break out of the loop at 302, which I think means we are still goign to have task_states be populated

Right, it will be populated with 'pending' - the initial state for the task, created at line 78 - and filtered out later on, producing no message